glue/sales_val_flagging.py

```diff
@@ -178,6 +178,7 @@ def classify_outliers(df, stat_groups: list, min_threshold):
         "sv_ind_ptax_flag_w_high_price_sqft": "High price per square foot",
         "sv_ind_price_low_price_sqft": "Low price per square foot",
         "sv_ind_ptax_flag_w_low_price_sqft": "Low price per square foot",
+        "sv_ind_raw_price_threshold": "Raw price threshold",
```

Comment:
Assigning this reason with priority right after the other price flags that are generated with the statistical flagging groups.

Order of assignment between the three outlier reasons will be

- Price outlier
- Raw price outlier (this cap)
- Ptax
- Other char reasons